Build fails on Travis

The build fails on Travis because of a change in the jquery externs file. The error is:

```
./node_modules/openlayers/node_modules/.bin/closure-util build build.json .build/build.js

info closure-util Reading build config

info closure-util Getting Closure dependencies

info closure-util Compiling 422 sources

ERR! compile /home/travis/build/Geoportail-Luxembourg/geoportailv3/geoportailv3/static/js/search/searchdirective.js:570: ERROR - Bad type annotation. Unknown type jQuery.event

ERR! compile  * @param {jQuery.event} event

ERR! compile            ^

ERR! compile 

ERR! compile 

ERR! compile 1 error(s), 0 warning(s)

ERR! compile 

ERR! closure-util Process exited with non-zero status, see log for more detail: 1

make: *** [.build/build.js] Error 1

The command "make -f travis.mk build" failed and exited with 2 during .

Your build has been stopped.
```

I'm looking into it just now…

--- a/geoportailv3/static/js/search/searchdirective.js
+++ b/geoportailv3/static/js/search/searchdirective.js
@@ -567,7 +567,7 @@ app.SearchDirectiveController.getAllChildren_ =
 
 
 /**
- * @param {jQuery.event} event
+ * @param {jQuery.Event} event
  * @param {(ol.Feature|Object|app.BackgroundLayerSuggestion)} suggestion
  * @param {string} dataset Typeahead dataset ID.
  * @this {app.SearchDirectiveController}

